setup.cfg

```diff
@@ -23,6 +23,12 @@ statistics = True
 [pylint]
 ignore = migrations,CVS
 
+[pylint.FORMAT]
+max-line-length = 120
+
+[pylint.messages_control]
+disable = C0103, W0707, R0903, R0904, W0511, R0901
```

Comment:
Same here. Can we be more descriptive using the error name instead of the code? I think it is better to know what we are disabling.
Are there some refactors we can apply to avoid disabling some errors? For example, invalid-name / C0103?
If we need to change many things and structures to avoid the error, we can disable it as you did, but if we can refactor it to avoid disabling it, it is better, to maintain and improve the quality of our work.

Are we using the messages control from the `[MESSAGES CONTROL]` part?

Reply:
Thanks, I made some adjustments